live-editing/configs/HierarchicalGridConfigGenerator.ts

```diff
@@ -993,9 +993,9 @@ export class HierarchicalGridConfigGenerator implements IConfigGenerator {
 
             ],
             appModuleConfig: new AppModuleConfig({
-                imports: ['HierarchicalGridValidatorServiceExtendedComponent', 'IgxHierarchicalGridModule', 'IgxPreventDocumentScrollModule'],
+                imports: ['HierarchicalGridValidatorServiceExtendedComponent', 'IgxHierarchicalGridModule', 'IgxPreventDocumentScrollModule', 'ReactiveFormsModule'],
```

Comment:
Shouldn't this be added in the TreeGridConfigGenerator for the cross-field validation sample config?

Reply:
Yup, I have misplaced it. Gonna fix that now.